src/css/watermark.css

```diff
@@ -1,11 +1,12 @@
-.md-header {
-    background-color: #ff0000;
-}
-
 .md-main__inner {
     background-image: url(../images/draft_watermark.png);
     background-repeat: no-repeat;
     background-attachment: fixed;
     background-size: contain, cover;
     background-position: center;
 }
+
+ /* See: https://squidfunk.github.io/mkdocs-material/setup/changing-the-colors/#custom-color-schemes */
+:root {
+    --md-primary-fg-color: #ff0000;
+}
```

Comment:
yeah, that works ... but it changes ALL previously blue text to red, including the links. 

I think it's okay, ... and I think I've had enough fiddling with this for this Saturday.

WDYT @effigies ?